<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
	 border-radius: 20px;
	background-color: #FF6347;
	border: 1px solid gray;
	padding: 5px;
	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QPushButton" name="Button6">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="Button3">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="Button8">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="MemAdd">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>M+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="Button1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="Button5">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QPushButton" name="Add">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QPushButton" name="MemGet">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>M</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="Multiply">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>*</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="Button9">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="Button4">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="MemClear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>M-</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="Button7">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="Divide">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="Button2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="Clear">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="Button0">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #FFB6C1;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="ChangeSign">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>+/-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QPushButton" name="Subtract">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="4" column="4">
     <widget class="QPushButton" name="Equals">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    border-radius: 15px; 
    background-color: #3498db;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}

QPushButton:pressed {
    border-radius: 15px;
    background-color: #FF1493;
    color: #000000;
    padding: 5px;
    border: 2px solid #2980b9;
}</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="5">
     <widget class="QLineEdit" name="Display">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
	background-color: #6B8E23;
	border: 1px solid gray;
	color: #000000;
}</string>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
